micro-ui/web/micro-ui-internals/packages/libraries/CHANGELOG.md

```
@@ -0,0 +1,47 @@
+# Changelog
+
+## [1.8.13]  [25-Apr-2025]
+- Updated Custom mutation hook and usecutsomAPI hook to handle header and method in request.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)